src/Base/Console.h

```diff
@@ -515,24 +515,32 @@ class BaseExport ILogger
  */
 class BaseExport ConsoleSingleton
 {
-
 public:
     static const unsigned int BufferSize = 4024;
     // exported functions goes here +++++++++++++++++++++++++++++++++++++++
     /// Prints a Message
-    virtual void Message ( const char * pMsg, ... );
+    void Message ( const char * pMsg, ... );

```

Comment:
Note:

Legacy Message/Warning/Error/Log functions stop being virtual. New MessageS/WarningS/ErrorS/LogS counterparts are created, which take an additional notifier parameter (also non-virtual). Both this are then handled via a virtual private interface (NVI pattern) for extension (see MessageV/WarningV/ErrorV/LogV).

The Console client user has then the option to emit regular legacy message (not identifying the notifier), or use the new versions identifying the notifier.